README.md

```diff
@@ -1,111 +1,170 @@
+
+### More about Sendbird Calls SDK for iOS
 
-## Functional Overview
-The SendBird Calls iOS SDK provides a framework to make and receive voice and video calls. “Direct calls” in the SDK refers to one-to-one calls, comparable to “direct messages” (DMs) in messaging services. To make a direct voice or video call, the caller specifies the user ID of the intended callee, and dials. Upon dialing, all of the callee’s authenticated devices will receive incoming call notifications. The callee then can choose to accept the call from any one of the devices. When the call is accepted, a connection is established between the caller and the callee. This marks the start of the direct call. Call participants may mute themselves, as well as select the audio and video hardware used in the call. Calls may be ended by either party. The SendBird Dashboard displays call logs in the Calls menu for application owners and admins to review.
+Find out more about Sendbird Calls for iOS on [Calls SDK for iOS doc](https://sendbird.com/docs/calls/v1/ios/getting-started/about-calls-sdk).
+
+<br />
+
+## Before getting started
+
+This section shows the prerequisites you need to check to use Sendbird Calls SDK for iOS.
+
+### Requirements
 
-## SDK Prerequisites
 * iOS 9.0 or later
 * Swift 4 or later, Objective-C
 * Xcode 9 or later, macOS Sierra or later.
 * Installation of **[Git Large File Storage](https://git-lfs.github.com)**
 
-> **IMPORTANT**: **MAKE SURE to install Git LFS before installing the pod**. Check the size of `WebRTC.framework` in SendBirdWebRTC folder. It MUST be over 800 MB. If the loaded SendBirdWebRTC framework is smaller than that, check the *Git Large File Storage* settings and download again. Refer to [the troubleshooting section](#Library-not-loaded-WebRTC.framework).
+> **IMPORTANT**: **MAKE sure to install Git LFS before installing the pod**. The size of `WebRTC.framework` in **SendBirdWebRTC** folder must be over 800 MB. If the size of the loaded `SendbirdWebRTC` framework is smaller than 800 MB, check the **Git Large File Storage** settings and download again. For further details, refer to SDK’s [troubleshooting section](#library-not-loaded-webrtcframework).
```

Comment:
The B in `SendbirdWebRTC` should be capitalized(`SendBirdWebRTC`) as it's a framework name.

Reply:
Done!